beetsplug/thumbnails.py

```diff
@@ -0,0 +1,109 @@
+# -*- coding: utf-8 -*-
+# This file is part of beets.
+# Copyright 2015, Bruno Cauet
+#
+# Permission is hereby granted, free of charge, to any person obtaining
+# a copy of this software and associated documentation files (the
+# "Software"), to deal in the Software without restriction, including
+# without limitation the rights to use, copy, modify, merge, publish,
+# distribute, sublicense, and/or sell copies of the Software, and to
+# permit persons to whom the Software is furnished to do so, subject to
+# the following conditions:
+#
+# The above copyright notice and this permission notice shall be
+# included in all copies or substantial portions of the Software.
+
+"""Create freedesktop.org-compliant thumnails for album folders"""
+
+from __future__ import (division, absolute_import, print_function,
+                        unicode_literals)
+
+from hashlib import md5
+import os
+import shutil
+from pathlib import PurePosixPath
+
+from xdg import BaseDirectory
```

Comment:
Yes it is. That's a oversight on my part. It could be used in `beets.util.confit` by the way, instead of digging in the environment variables and having hard-coded fallbacks (which XDG provides).